### PR TITLE
Code of conduct and changelog

### PR DESCRIPTION
**Pull request by franciscoj**
### :tophat: Wat?

Add the Contributor Covenant because it encourages a better community http://contributor-covenant.org/

Add a changelog because it helps with the gem usage a lot http://keepachangelog.com/

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+By adopting this Code of Conduct, project maintainers commit themselves to
+fairly and consistently applying these principles to every aspect of managing
+this project. Project maintainers who do not follow or enforce the Code of
+Conduct may be permanently removed from the project team.
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting a project maintainer at support@redbooth.com. All
```
Comment (franciscoj) [inline CODE_OF_CONDUCT.md]: Which email address should we use here?

**Comment by sauloperez** [comment]
Cool! please, add yourself as gem author!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (sauloperez) [inline CODE_OF_CONDUCT.md]: better to accounts at redbooth

**Comment by csaura** [comment]
:+1:

**Comment by jhbabon** [comment]
Cool. Every maintainer should read the code of conduct an be aware of it. I see that the contact email is `accounts@redbooth.com`. Is that email right? Do we want another email for open source projects?

**Comment by jhbabon** [comment]
And :+1:

**Comment by sauloperez** [comment]
It's fine. We just changed it to accounts so that it gets read by devs.

**Pull request merged**